src/runtime/importhook.cs

```diff
@@ -58,13 +57,12 @@ internal static void Initialize()
             clr_dict = (IntPtr)Marshal.PtrToStructure(clr_dict, typeof(IntPtr));
 
             Runtime.PyDict_Update(mod_dict, clr_dict);
-            Runtime.PyDict_SetItemString(dict, "CLR", py_clr_module);
-            Runtime.PyDict_SetItemString(dict, "clr", py_clr_module);
 #elif PYTHON2
             Runtime.XIncref(root.pyHandle); // we are using the module two times
-            Runtime.PyDict_SetItemString(dict, "CLR", root.pyHandle);
-            Runtime.PyDict_SetItemString(dict, "clr", root.pyHandle);
+            py_clr_module = root.pyHandle;  // Alias handle for PY2/PY3
```

Comment:
@filmor Since you just worked on this, any reason this wouldn't work or is a bad idea?